src/tasks/acceptability.py

```diff
@@ -0,0 +1,433 @@
+"""Define the tasks and code for loading their data.
```

Comment:
This comment seems wrong. Did you just copy it from somewhere else? It need to explain what this file is for, and when someone should use it.
 
Also, if CoLA isn't here, 'acceptability' isn't a good name for the file. 'acceptability_probing.py'?

Reply:
(1) Exactly.
(2) Sure.